UITests/AuthenticationManagerTests.swift

```diff
@@ -148,14 +122,11 @@ class AuthenticationManagerTests: KIFTestCase {
     }
 
     func testChangeRequirePasscodeInterval() {
-        setPasscode("1337", interval: .Immediately)
+        PasscodeUtils.setPasscode("1337", interval: .Immediately)
 
         openAuthenticationManager()
         tester().tapViewWithAccessibilityLabel("Require Passcode, Immediately")
 
-        tester().waitForViewWithAccessibilityLabel("Enter Passcode")
-        enterPasscodeWithDigits("1337")
-
```

Comment:
I never noticed it before but I ran the current production version (3.0) and I notice that we always load the home panels on fresh launch. We never ran into this issue because the elements we were 'tapping' were ready to go - like the URL bar and show tabs button. So I guess it actually is in a loading state. I think your PR will resolve this.